homeassistant/components/locative/__init__.py

```diff
@@ -28,6 +28,7 @@
 
 ATTR_DEVICE_ID = 'device'
 ATTR_TRIGGER = 'trigger'
+ATTR_DEVICE_TYPE = 'device_type'
```

Comment:
This slipped in :-)

Reply:
Thanks! I was debating adding it (since it was only one attribute that was missing and then I discovered there were several more) and that it didn't make sense.